modules/player/src/main/java/unnamed/mmo/player/PlayerImpl.java

```diff
@@ -27,6 +33,8 @@ public class PlayerImpl extends Player {
     private int diggingBlockMaxHealth = 0;
     private BlockFace diggingFace = null;
 
+    private final Map<String, ModifierList> currentModifiers = new HashMap<>();
```

Comment:
There is a missing way to get a value for a given modifier type?

Reply:
There is, with `currentModifiers.get(modifierType).calculateTotal();`, but I have added a helper method for this